src/arrayMethodSort.js

```diff
@@ -4,8 +4,29 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    for (let i = 0; i < this.length; i++) {
+      for (let k = 0; k < this.length; k++) {
+        const comparedEl = this[i];
+        const currentEl = this[k];
+
+        if (compareFunction === undefined) {
+          if (this[i].toString() < this[k].toString()) {
+            this[i] = currentEl;
+            this[k] = comparedEl;
+          }
```

Comment:
To swap elements, it's more common and readable to use array destructuring like `[this[k], this[i]] = [this[i], this[k]]`. This avoids the need for temporary variables and makes the code cleaner.